diagnostic.md

```diff
@@ -16,13 +16,13 @@ Graces-MacBook-Pro:try grace$
 
 What is the full path to the current directory?
 
-_Replace this text with your answer._
+Graces-MacBook-Pro:
 
 ## Question 2
 
 Describe GitHub, **briefly** in your own words.
 
-_Replace this text with your answer._
+Github is a way to show your coding process
```

Comment:
Not quite. GitHub is a user interface for Git. Git is a powerful tool for "version control" - it means that project files can be updated without creating multiple copies, and they can also be shared on the internet.